test/functional/api/vim_spec.lua

```diff
@@ -393,7 +393,7 @@ describe('API', function()
          string.match(rv, "E%d*:.*"))
       eq('', eval('v:errmsg'))  -- v:errmsg was not updated.
       -- Verify NO hit-enter prompt.
-      eq({mode='n', blocking=false}, nvim("get_mode"))
+      eq({mode='n', blocking=false, cmdpreview=false}, nvim("get_mode"))
```

Comment:
why isn't it a subflag of `mode`? (we could, as a separate effort, expose those flags in a "structured" way--but that is a separate topic)

Reply:
Done, it's a subflag of mode now :)